app/javascript/components/projects/ProjectView.jsx

```diff
@@ -144,50 +149,92 @@ class ProjectView extends React.Component {
             <div className="tl fl w-75 ml6 mr6 mt6 mb5 bg-white pa5">
                 <h1 className="f1 ma0">{project.title}</h1>
                 {edit_button}
-                <div className="dib rt-yellow-bg ph3 pv2 mv4 f5 fw4">
-                    {project_string_status}
+                <div>
+                    <div className="dib ph3 ba pv2 mv3 f5 fw4">
+                        {project_string_status} 
+                    </div>
+                    {apply_button}
                 </div>
-                <h3 className="mt3">Project Overview</h3>
-                <p>{project.overview}</p>
+                <img className="mv3 w-100 h-auto" src={project_pic}/>
+                {project.overview ? (
+                            <div>
+                                <h3 className="mt3 f3">Project Overview</h3>
+                                <p className="f5">{project.overview}</p>
+                            </div>
+                        ) : (null)}
                 <div className="mt3 flex items-start">
                     <div className="w-75">
-                        <h3 className="mt3">Description</h3>
-                        <p>{project.overview}</p>
-                        <h3 className="mt3">Deliverable</h3>
-                        <p>{project.deliverable}</p>
-                        <h3 className="mt3">Volunteer Requirements</h3>
-                        <p>{project.volunteer_requirements}</p>
-                        <h3 className="mt3">Other Details</h3>
-                        <p>{project.other_details}</p>
-                        <h3 className="mt3">Our Community Needs This If</h3>
-                        <p>{project.question1}</p>
-                        <h3 className="mt3">The Right Volunteer for this Project Is</h3>
-                        <p>{project.question2}</p>
-                        <h3 className="mt3">What You Give, What You Get</h3>
-                        <p>{project.question3}</p>
+                        {project.description ? (
+                            <div>
+                                <h3 className="mt4 f3">Description</h3>
+                                <p className="f5">{project.description}</p>
+                            </div>
+                        ) : (null)}
+                        {project.deliverable ? (
+                            <div>
+                                <h3 className="mt4 f3">Deliverable</h3>
+                                <p className="f5">{project.deliverable}</p>
+                            </div>
+                        ) : (null)}
+                        {this.props.skills ? (
+                            <div>
+                                <h3 className="mt4 f3">Volunteer Skills</h3>
+                                <p className="f5">{this.props.skills}</p>
```

Comment:
Since this will be a list of objects, you should probably do something where you loop through each skill and render it a certain way, similar to how we render applications. e.g. `applicationList = this.props.applications.map((application, index) => {...`